web/cypress/integration/route/online-debug.spec.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+import menuLocaleUS from '../../../src/locales/en-US/menu';
+import routeLocaleUS from '../../../src/pages/Route/locales/en-US';
+import defaultSettings from '../../../config/defaultSettings';
```

Comment:
in order to reuse the definition, I imported the locales files instead of using the text of the dom. 

cc @juzhiyuan @LiteSun